### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by AryanParashar24**
Added 2 Book for Go for the introducing go and concurrency in Go in the Go Section

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by sohansai — COMMENTED** [review]
Access to the resources is not free.

Include freely available resources. Refer to docs/CONTRIBUTING.md for contribution guidelines."

**Comment by AryanParashar24** [comment]
@sohansai Sry for the previous inclusions plz check out the updates now

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -816,6 +816,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [An Introduction to Programming in Go](https://www.golang-book.com/books/intro) - Caleb Doxsey
 * [Build Web Application with Golang](https://astaxie.gitbooks.io/build-web-application-with-golang/content/en/) - astaxie
 * [Building Web Apps with Go](https://codegangsta.gitbooks.io/building-web-apps-with-go/content/)
+* [Concurrency in Go](https://edu.anarcho-copy.org/Programming%20Languages/Go/Concurrency%20in%20Go.pdf) - Katherine Cox-Buday (PDF)
```
Comment (sohansai) [inline books/free-programming-books-langs.md:819]: This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -832,6 +833,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Golang by Example](https://golangbyexample.com)
 * [Golang tutorial series](https://golangbot.com/learn-golang-series/) - Naveen Ramanathan
 * [How To Code in Go](https://www.digitalocean.com/community/books/how-to-code-in-go-ebook) - Mark Bates, Cory Lanou, Timothy J. Raymond (PDF, EPUB)
+* [Introducing Go](https://library.kre.dp.ua/Books/2-4%20kurs/%D0%9F%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D1%83%D0%B2%D0%B0%D0%BD%D0%BD%D1%8F%20%2B%20%D0%BC%D0%BE%D0%B2%D0%B8%20%D0%BF%D1%80%D0%BE%D0%B3%D1%80%D0%B0%D0%BC%D1%83%D0%B2%D0%B0%D0%BD%D0%BD%D1%8F/Go%20Web/Introducing%20Go%40bzd_channel.pdf) - Caleb Doxsey (PDF)
```
Comment (sohansai) [inline books/free-programming-books-langs.md:836]: This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.

**Pull request closed without merging**